ale/drivers/mro_drivers.py

```diff
@@ -344,23 +343,6 @@ def sensor_name(self):
         """
         return "CONTEXT CAMERA"
 
-    @property
```

Comment:
This seems a weird change to include in the PR? CHANGELOG entry might clear up if this is a bug fix for something else or what.